### ISSUE TITLE
This project is probably supplanted by PEP 526

### ISSUE THREAD

**Issue by NeilGirdhar** [eugene-eeo/promises#2]

**Comment by eugene-eeo** [comment]
True, but this is more of a proof-of-concept and my experiment to expose a DSL-like API. FWIW it's no longer actively maintained (unless I come back to it in the future).
